cluster-provision/cli/cli

```diff
@@ -173,7 +173,13 @@ if [ "$COMMAND" = "update" ] ; then
   exit 0
 fi
 
-DNSMASQ_CID=$(docker run -d ${PORTS} -v /lib/modules:/lib/modules -e NUM_NODES=${NODES} --name ${PREFIX}dnsmasq --privileged ${BASE} /bin/bash -c /dnsmasq.sh)
+# gocli will try to mount /lib/modules to make it accessible to dnsmasq in
+# in case it exists
+if [ -f /lib/modules ]; then
+        dnsmasq_mounts="-v /lib/modules/:/lib/modules/"
```

Comment:
i would check if ip6tables command can be executed (which means the module already installed), and if it doesnt i will exit 1
because else the provision would failed on the errors that dnsmasq cant execture the ip6tables commands

Reply:
In case /lib/modules is not there it's suppose to be already loaded (that's the case for CI) we can document that but that's exactly what we want for CI since CI is not going to mount /lib/modules as this is a security issue.